src/driver/test/CrudActor.yml

```diff
@@ -0,0 +1,38 @@
+SchemaVersion: 2018-07-01
+  
+Actors:
+- Name: CrudActor
+  Type: CrudActor
+  Database: mydb
+  ExecutionStrategy:
+    ThrowOnFailure: true
+  Phases:
+  - Repeat: 10
+    Collection: test
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: insertOne
+          Document: { a: 1 }
+        - WriteCommand: updateOne
+          Filter: { a: 1 }
+          Update: { $set: { a: {^RandomInt: {min: 5, max: 15} } } }
+          Options:
+            WriteConcern:
+              Level: majority
+              TimeoutMillis: 5000
+    - OperationName: insertMany
+      OperationCommand:
+        Documents:
+        - { a: 1 }
+        - { a : {^RandomString: {length: {^RandomInt: {min: 3, max: 5} }}} }
+        - { b : {^RandomInt: {min: 5, max: 15} } }
+  - Repeat: 1
+    Collection: test
+    Operation:
+      OperationName: drop
+      OperationCommand:
+        Options:
+          WriteConcern:
+            Level: majority
```

Comment:
I like the example you had in the class docs - can you put that here (or in a separate yaml file) and then just comment in the class docs to see example in the file?